Answers.md

```diff
@@ -0,0 +1,10 @@
+1. A database is a set of data held in computer that is accessible in various ways. Collections are analagous to tables in relational databases. MongoDB stores data as BSON documents. MongoDB documents are composed of field-and-value pairs with the following structure:
+{
+	field1: value1,
+	field2: value2,
+	field3: value3	
+}
+
+2. One can achieve a pattern of relationships either by using embedded documents to describe one-to-one and one-to-many relationships between connected data, or by using references to describe one to many relationships between documents. At the schema level, data can be embedded as an array to optimze performance. Need help with the rest of this question.
```

Comment:
At the schema level you can use refs and objectIds that relate to another Schema.  You can "fill in" this data using populate.